sktime/utils/plotting.py

```diff
@@ -91,6 +93,25 @@ def plot_series(*series, labels=None, markers=None):
 
         plot_func(x=x, y=y, ax=ax, marker=marker, label=label, color=color)
 
+    # plot prediction intervals if present
+    if pred_int is not None:
+        # check same conditions as for earlier indices
+        if not type(index) is type(pred_int.index):  # noqa
```

Comment:
pls use here the existing function `check_equal_time_index` instead, where you can pass multiple `pd.Series` to check for same index. So you can just take the first column of `X` to get a `pd.Series` with the index. Also you could use `check_X_y` by adding `X, y = check_X_y(X, y)`